STYLE_GUIDE.md

```diff
@@ -272,6 +272,20 @@ end
   So set up it on the local rubocop config manually.
   <sup>[[link](#style-magic-link)]</sup>
 
+* <a name="style-braces-around-hash-params"></a>
+  Write empty methods in an expanded way.
```

Comment:
Looks like somebody forgot to change the copied-pasted text 😉

Reply:
Oops